### ISSUE TITLE
Crash on objc_getClassList when run UnitTest

### ISSUE THREAD

**Issue by bupojung** [Quick/Quick#910]
### What did you do?

Please replace this with what you did.

### What did you expect to happen?

Please replace this with what you expected to happen.

### What actually happened instead?

Please replace this with what happened instead.

### Environment

List the software versions you're using:

 - Quick: 2.1.0
 - Nimble: 8.0.1
 - Xcode Version: 10.2.1 
 - Swift Version: 5 

Please also mention which package manager you used and its version. Delete the
other package managers in this list:

 - Cocoapods:1.6.2

### Project that demonstrates the issue
backtrace:

```
thread #1, queue = 'com.apple.main-thread', stop reason = EXC_BAD_ACCESS (code=1, address=0x9)
    frame #0: 0x0000000129daa02f libswiftCore.dylib`swift_checkMetadataState + 383
    frame #1: 0x0000000129dadcd6 libswiftCore.dylib`bool findAnyTransitiveMetadata<checkTransitiveCompleteness(swift::TargetMetadata<swift::InProcess> const*)::$_14&>(swift::TargetMetadata<swift::InProcess> const*, checkTransitiveCompleteness(swift::TargetMetadata<swift::InProcess> const*)::$_14&) + 118
    frame #2: 0x0000000129dadbbe libswiftCore.dylib`checkTransitiveCompleteness(swift::TargetMetadata<swift::InProcess> const*) + 110
  * frame #3: 0x0000000129dae9b8 libswiftCore.dylib`swift::MetadataCacheEntryBase<(anonymous namespace)::SingletonMetadataCacheEntry, int>::doInitialization(swift::ConcurrencyControl&, swift::MetadataCompletionQueueEntry*, swift::MetadataRequest) + 264
    frame #4: 0x0000000129da4213 libswiftCore.dylib`swift_getSingletonMetadata + 579
    frame #5: 0x0000000148f74928 XXXTests`type metadata accessor for xxxCellViewModel at <compiler-generated>:0
    frame #6: 0x0000000148f74fec XXXTests`ObjC metadata update function for xxxCellViewModel at <compiler-generated>:0
    frame #7: 0x0000000127d3cd7f libobjc.A.dylib`realizeClassMaybeSwiftMaybeRelock(objc_class*, mutex_tt<false>&, bool) + 126
    frame #8: 0x0000000127d33b98 libobjc.A.dylib`realizeAllClasses() + 97
    frame #9: 0x0000000127d358f6 libobjc.A.dylib`objc_getClassList + 62
    frame #10: 0x0000000149d123c0 XXXTests`qck_enumerateSubclasses(klass=QuickConfiguration, block=0x0000000149d126c0) at QuickConfiguration.m:21:24
    frame #11: 0x0000000149d12673 XXXTests`__32+[QuickConfiguration initialize]_block_invoke(.block_descriptor=0x000000014a3a2fb0) at QuickConfiguration.m:74:13
    frame #12: 0x000000012a756d02 libdispatch.dylib`_dispatch_client_callout + 8
    frame #13: 0x000000012a7584ce libdispatch.dylib`_dispatch_once_callout + 20
    frame #14: 0x0000000149d12615 XXXTests`+[QuickConfiguration initialize] [inlined] _dispatch_once(predicate=0x000000014a674740, block=0x0000000149d12640) at once.h:84:3
    frame #15: 0x0000000149d125fa XXXTests`+[QuickConfiguration initialize](self=QuickConfiguration, _cmd="initialize") at QuickConfiguration.m:73
    frame #16: 0x0000000127d30698 libobjc.A.dylib`CALLING_SOME_+initialize_METHOD + 19
    frame #17: 0x0000000127d30a2a libobjc.A.dylib`initializeNonMetaClass + 269
    frame #18: 0x0000000127d3154f libobjc.A.dylib`initializeAndMaybeRelock(objc_class*, objc_object*, mutex_tt<false>&, bool) + 144
    frame #19: 0x0000000127d373e2 libobjc.A.dylib`lookUpImpOrForward + 205
    frame #20: 0x0000000127d450d4 libobjc.A.dylib`_objc_msgSend_uncached + 68
    frame #21: 0x0000000149d1283f XXXTests`+[QuickSpec initialize](self=QuickSpec, _cmd="initialize") at QuickSpec.m:29:5
    frame #22: 0x0000000127d30698 libobjc.A.dylib`CALLING_SOME_+initialize_METHOD + 19
    frame #23: 0x0000000127d30a2a libobjc.A.dylib`initializeNonMetaClass + 269
    frame #24: 0x0000000127d30969 libobjc.A.dylib`initializeNonMetaClass + 76
    frame #25: 0x0000000127d3154f libobjc.A.dylib`initializeAndMaybeRelock(objc_class*, objc_object*, mutex_tt<false>&, bool) + 144
    frame #26: 0x0000000127d373e2 libobjc.A.dylib`lookUpImpOrForward + 205
    frame #27: 0x0000000127d450d4 libobjc.A.dylib`_objc_msgSend_uncached + 68
    frame #28: 0x0000000124d42d41 Foundation`-[NSBundle loadAndReturnError:] + 1359
    frame #29: 0x00000001220cb743 libXCTestBundleInject.dylib`__XCTestBundleInject + 630
    frame #30: 0x0000000122052b96 dyld_sim`ImageLoaderMachO::doModInitFunctions(ImageLoader::LinkContext const&) + 506
    frame #31: 0x0000000122052d9c dyld_sim`ImageLoaderMachO::doInitialization(ImageLoader::LinkContext const&) + 40
    frame #32: 0x000000012204e3fc dyld_sim`ImageLoader::recursiveInitialization(ImageLoader::LinkContext const&, unsigned int, char const*, ImageLoader::InitializerTimingList&, ImageLoader::UninitedUpwards&) + 324
    frame #33: 0x000000012204d5d3 dyld_sim`ImageLoader::processInitializers(ImageLoader::LinkContext const&, unsigned int, ImageLoader::InitializerTimingList&, ImageLoader::UninitedUpwards&) + 133
    frame #34: 0x000000012204d665 dyld_sim`ImageLoader::runInitializers(ImageLoader::LinkContext const&, ImageLoader::InitializerTimingList&) + 73
    frame #35: 0x0000000122041333 dyld_sim`dyld::initializeMainExecutable() + 129
    frame #36: 0x0000000122045434 dyld_sim`dyld::_main(macho_header const*, unsigned long, int, char const**, char const**, char const**, unsigned long*) + 4384
    frame #37: 0x0000000122040630 dyld_sim`start_sim + 136
    frame #38: 0x0000000123c1d234 dyld`dyld::useSimulatorDyld(int, macho_header const*, char const*, int, char const**, char const**, char const**, unsigned long*, unsigned long*) + 2238
    frame #39: 0x0000000123c1b0ce dyld`dyld::_main(macho_header const*, unsigned long, int, char const**, char const**, char const**, unsigned long*) + 522
    frame #40: 0x0000000123c16503 dyld`dyldbootstrap::start(macho_header const*, int, char const**, long, macho_header const*, unsigned long*) + 1167
    frame #41: 0x0000000123c16036 dyld`_dyld_start + 54
```

**Comment by achernoprudov** [comment]
Related to #912.

**Comment by ikesyo** [comment]
I believe this is already addressed by #873 and #901.
